specification/dartLangSpec.tex

```diff
@@ -5998,6 +5998,27 @@ \subsubsection{The Instantiation to Bound Algorithm}
 also when they are deeply nested.%
 }
 
+\LMHash{}%
+\IndexCustom{Instantiation to bound on a generic function $f$}{%
+  generic function!instantiation to bound}
+also uses the algorithm described above,
+taking the formal parameters \List{X}{1}{k} from the declaration of $f$,
+with bounds \List{B}{1}{k}, and,
+for each $i \in 1 .. k$,
+letting $S_i$ denote the result of instantiation to bound on $B_i$,
+and letting $S_i$ be \DYNAMIC{} when the $i$th bound is omitted.
+
+\LMHash{}%
+Let $f$ be a generic function declaration.
+If instantiation to bound on $f$ yields
+a list of type arguments \List{T}{1}{k} such that,
+for some $j \in 1..k$,
+$T_j$ is or contains a type which is not well-bounded,
+or if \List{T}{1}{k} does not satisfy their bounds,
```

Comment:
It's a little confusing to say "satisfy their bounds", since the bounds in question are not a property of the type arguments but of the function.  Consider re-phrasing as "does not satisfy the bounds on the formal parameters to `f`" or something similar?

Reply:
Indeed, done!